src/Tuura/Concept/Circuit/Derived.hs

```diff
@@ -171,6 +183,19 @@ xorGate :: a -> a -> a -> CircuitConcept a
 xorGate i1 i2 o = [rise i1, rise i2] ~|~> rise o <> [fall i1, fall i2] ~|~> rise o
                <> [rise i1, fall i2] ~|~> fall o <> [fall i1, rise i2] ~|~> fall o
 
+-- Set/reset latch including one non-inverted and one inverted output.
+srLatch :: Eq a => a -> a -> a -> a -> CircuitConcept a
+srLatch s r q nq = never [rise s, rise r]      -- disallow contradictory requests
+    <> initialise0 [q]   <> initialise1 [nq]   -- disallow initial state q=nq=0
```

Comment:
I think a better comment here would be `set initial state to q=0 and nq=1`. And add the following on top of the function:

`TODO: Generalise by allowing both 01 and 10 as initial states.`